src/docs/blueprint_download.md

```diff
@@ -30,7 +30,7 @@ Eclipse Theia Blueprint is ***not*** **a production-ready product**. Therefore,
     <tr>
       <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/latest/windows/TheiaBlueprint.exe&r=1" download>Download latest</a></td>
       <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/latest/linux/TheiaBlueprint.AppImage&r=1" download>Download latest</a></td>
-      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/latest/macos/TheiaBlueprint.pkg&r=1" download>Download latest</a></td>
+      <td><a href="https://www.eclipse.org/downloads/download.php?file=/theia/latest/macos/TheiaBlueprint.dmg&r=1" download>Download latest</a></td>
```

Comment:
@jfaltermeier The download link does not work for me (https://deploy-preview-164--romantic-mcclintock-8d974f.netlify.app/)

Reply:
See comment above: -> Website needs to be updated after eclipse-theia/theia-blueprint#94 was merged and the dmg was published for the first itme

Reply:
got it, sorry!

Reply:
It's merged an published now, so I'll update the website